pyhuman/app/workflows/open_office_writer.py

```diff
@@ -0,0 +1,141 @@
+        super().__init__(name=WORKFLOW_NAME, description=WORKFLOW_DESCRIPTION)
+        self.default_wait_time = default_wait_time
+        self.open_office_path = open_office_path
+
+    def action(self, extra=None):
+        self._create_document()
+
+    def _create_document(self):
+        self._new_document()
+        # Type random paragrahs and sentences
+        for i in range(0, random.randint(2,10)):
+            random.choice([pyautogui.typewrite(TextLorem().paragraph()), pyautogui.typewrite(TextLorem().sentence())])
+            pyautogui.press('enter')
+        sleep(self.default_wait_time)
+        # Randomly perform actions
+        for i in range(0, random.randint(6,15)):
+            random.choice([
+                self._save_pdf,
+                self._write_sentence,
+                self._write_paragraph,
+                self._copy_paste, 
+                self._insert_comment,
+                self._find,
+                self._delete_text,
+                self._format_text])()
+            sleep(self.default_wait_time)
+        # Save and quit the document
+        self._save_quit()
+
+    def _insert_comment(self):
+        pyautogui.hotkey('ctrl', 'alt', 'c') # insert comment
+        pyautogui.typewrite(TextLorem().sentence()) # type random sentence
+        pyautogui.press('esc') # finish commenting
+        sleep(self.default_wait_time)
+
+    def _find(self):
+        pyautogui.hotkey('ctrl', 'f') # open Find & Replace
+        sleep(self.default_wait_time)
+        pyautogui.typewrite(TextLorem()._word()) # type random word
+        sleep(self.default_wait_time)
+        pyautogui.press('enter') 
+        sleep(self.default_wait_time)
+        pyautogui.hotkey('alt','y') # close pop up box that may appear
+        sleep(self.default_wait_time)
+        pyautogui.hotkey('alt','c') # close Find & Replace
+        sleep(self.default_wait_time)
+
+    def _copy_paste(self):
+        self._select_text()
+        sleep(self.default_wait_time)
+        pyautogui.hotkey('ctrl', 'c') # copy to clipboard
+        sleep(self.default_wait_time)
+        pyautogui.press('backspace') # delete text
+        sleep(self.default_wait_time)
+        pyautogui.typewrite(TextLorem().paragraph()) # write text
+        sleep(self.default_wait_time)
+        pyautogui.press('enter') # insert new line
+        pyautogui.press('enter') # insert new line
+        pyautogui.hotkey('ctrl', 'v') # paste from clipboard
+        sleep(self.default_wait_time)
+
+    def _select_text(self):
+        selection_params = [
+            ['ctrl'  , 'home'], # go to beginning of document
+            ['shift' , 'left'], # move cursor & select to left
+            ['shift' , 'up'] # move cursor & select up
+        ]
+        pyautogui.hotkey(*random.choice(selection_params)) 
+
+    def _format_text(self):
+        self._select_text()
+        sleep(self.default_wait_time)
+        formatting_params = [
+        ['ctrl','1'], # Apply heading 1 style
```

Comment:
Another small formatting nitpick, but it may look better with these list elements tabbed over.